lib/cocoapods/bazel.rb

```diff
@@ -135,18 +135,15 @@ def self.format_files(build_files:, buildifier:, workspace:)
       when true
         return unless Pod::Executable.which('buildifier')
 
-        args = ['buildifier']
+        args = %w[buildifier -type build] + build_files.each_key.map { |d| File.join workspace, d, 'BUILD.bazel' }
       when String, Array
         args = Array(buildifier)
       else
         return
       end
-      args += %w[-type build]
 
       executable, *args = args
-      Pod::Executable.execute_command executable,
```

Comment:
It looked like it results in different behavior when specifying buildifier as an array vs true?
1. `true`: format only governed BUILD files
2. `/path/to/buildifier`: formats the entire directory: no way to specify `cocoapods-bazel` only ones.

Consider not moving the part of the command `.each_key.map { |d| File.join workspace, d, 'BUILD.bazel' }` if so, this seems hard to reason about. I'd suggest to align them to do the same thing. It's nice that it doesn't try to mutate other files today.

I assumed what you wanted to do is pass `bazel run :buildifier --` - the key bit adding `--` so bazel passes arguments to the binary vs formatting the entire repo

Reply:
🙏🏼  `--` 🙏🏼